docs/cloud/provision/azure/considerations.md

```diff
@@ -1,31 +1,30 @@
+We're aware that Azure Premium SSD volumes have a strict IOPS limit and this limit depends on the volume size. Very small volumes do not provide enough throughput for most production scenarios. We suggest you consider using at least 246 GiB disks to get enough IOPS for the database. You can check the current throughput limits for Azure Premium SSD volumes in [Azure documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/disks-types#premium-ssd).
 
 ## Resource provisioning
 
 Azure resource provisioning speed will almost always be slower and more highly variable than other providers. Please account for this when interacting with provisioned resources on Azure.
 
 ## Performance and reliability
 
-During our tests and usage, Azure has proved to be the least reliable of all the support cloud providers. We have noticed substandard network performance and substandard disk IO performance when compared to AWS and GCP.
+During our tests and usage, AWS and GCP have proved to be more reliable support cloud providers. We have noticed AWS and GCP provide superior network performance and sufficient disk IO performance when compared to Azure, so this should be considered when choosing your cloud support provider. 
 
-Many Azure regions do not have the minimum number of availability zones required to support the deployment of managed EventStoreDB clusters. At the moment, you can only deploy EventStoreDB clusters in regions with three or more availability zones. In the future, we plan to allow the deployment of single node topologies to all regions. We'll also introduce a new topology that allows cluster deployments to a single availability zone.
+At the moment, you can deploy EventStoreDB clusters only in regions with three or more availability zones, and many Azure regions do not have the minimum number of availability zones to support the deployment of managed EventStoreDB clusters. In the future, we plan to allow the deployment of single node topologies to all regions. We'll also introduce a new topology that allows cluster deployments to a single availability zone.
 
-## Pricing
 
-Prices for compute resources (virtual machines) in Azure are generally higher, compared with AWS and GCP. Therefore, you will find that managed EventStoreDB deployments in Azure are more expensive.
+## Pricing
 
-In particular, the `F1` instance size in Azure is much more expensive than the same size in AWS and GCP. It's because we require at least two CPU cores and the smallest suitable instance size in Azure is significantly more expensive, compared to other cloud providers.
+Prices for computing resources (virtual machines) in Azure are generally higher, compared with AWS and GCP, and this is reflected in the costs for managed EventStoreDB deployments. As EventStoreDB requires at least two CPU cores and the smallest instance size in Azure is more expensive, the `F1` instance size in Azure is the most expensive of the three cloud providers we offer.  
```

Comment:
Should we even mention this ?
The pricing engine will show it to users .
and does the price make _any_ difference when, in general the choice of cloud provider is independent of EventStore and in the hand of users. We also don't have the full reasoning behind the choice of cloud provider.  
In other words , who are we to question their choice in that regards ?

Reply:
I had this thought when editing the page. Whilst it's good to explain all the negatives, this is a negative the reader may not be able to do anything about at this point. Keeping this point on the page makes it a useful piece of evidence as to why Azure shouldn't be chosen by the deciders, but I think any dev would come across this page too late to use it as evidence in the decision-making process for cloud providers.

Reply:
I think the reason this is in is because we've been asked why azure is more expensive in the past

Reply:
I think that it's worth mentioning and explaining. Especially, that user may be surprised why we are charging more for Azure and wondering if it's our margin or limitations.